src/main/java/org/folio/innreach/batch/contribution/service/ContributionJobRunner.java

```diff
@@ -318,22 +320,27 @@ public void cancelInitialContribution(UUID contributionId) {
   }
 
   private boolean isEligibleForContribution(UUID centralServerId, Instance instance) {
+    log.info("isEligibleForContribution:: parameters centralServerId: {}, instance id: {}", centralServerId, instance.getId());
```

Comment:
I think this line and the following log statements on all of these methods that return a boolean is excessive. What we might care about is whether or not the boolean was true, not that the method was called, which it always will be.

Reply:
Added Boolean log wherever isEligibleForContribution is called.